lib/sportradar/api/nfl/game.rb

```diff
@@ -19,7 +19,11 @@ def initialize(data)
         @utc_offset = data["utc_offset"]
         @weather = data["weather"]
         @clock = data["clock"]
-        @quarter = data["quarter"]
+        if data["quarter"]
+          @quarter = data["quarter"][0]
+          quarter_data = data["quarter"][1].is_a?(Hash) ? [ data["quarter"][1] ] : data["quarter"][1]
+          @quarters = quarter_data&.map { |hash| Sportradar::Api::Nfl::Quarter.new(hash) }
```

Comment:
Just pulled your code and ran it on a past game, only one quarter is being put into the quarters hash due to ["quarter"][1]:
```
past_game_id = "0141a0a5-13e5-4b28-b19f-0c3923aaef6e"
s = Sportradar::Api::Nfl.new('o').play_by_play past_game_id
s.quarters.count
# => 1
s.quarters.map(&:sequence) 
# => ["1"]

s.response['quarter'].map(&:class) 
# => [String, Hash, Hash, Hash, Hash]
```
I'll push a commit here, as I'm already here in the code